drivers/flash/spi_nor.c

```diff
@@ -1013,34 +1007,15 @@ static int setup_pages_layout(const struct device *dev)
  */
 static int spi_nor_configure(const struct device *dev)
 {
-	struct spi_nor_data *data = dev->data;
-	uint8_t jedec_id[SPI_NOR_MAX_ID_LEN];
 	const struct spi_nor_config *cfg = dev->config;
+	uint8_t jedec_id[SPI_NOR_MAX_ID_LEN];
 	int rc;
 
-	data->spi = device_get_binding(DT_INST_BUS_LABEL(0));
-	if (!data->spi) {
+	/* Validate bus is ready */
```

Comment:
The comment is a bit misleading since spi_is_ready() also checks the chip select device.